.ci/jobs.t/elastic+elasticsearch+pull-request+build-benchmark.yml

```diff
@@ -0,0 +1,51 @@
+---
+- job:
+    name: "elastic+elasticsearch+pull-request+build-benchmark"
+    display-name: "elastic / elasticsearch - pull request build benchmark"
+    description: "Testing of Elasticsearch pull requests - build benchmark"
+    workspace: "/dev/shm/elastic+elasticsearch+pull-request+build-bench"
+    scm:
+      - git:
+          refspec: "+refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*"
+          branches:
+            - "${ghprbActualCommit}"
+    properties:
+      - inject:
+          properties-content: |
+            BUILD_PERFORMANCE_TEST=true
+            COMPOSE_HTTP_TIMEOUT=120
+            JOB_BRANCH=%BRANCH%
+            HOME=$JENKINS_HOME
+            GRADLEW=./gradlew --parallel --scan --build-cache -Dorg.elasticsearch.build.cache.url=https://gradle-enterprise.elastic.co/cache/
+            GRADLEW_BAT=./gradlew.bat --parallel --scan --build-cache -Dorg.elasticsearch.build.cache.url=https://gradle-enterprise.elastic.co/cache/
+      triggers:
+        - github-pull-request:
+            org-list:
+              - elastic
+            allow-whitelist-orgs-as-admins: true
+            trigger-phrase: '.*run\W+elasticsearch-ci/build-bench.*'
+            github-hooks: true
+            status-context: elasticsearch-ci/build-benchmark
+            cancel-builds-on-update: true
+            black-list-target-branches:
+              - 6.8
+            excluded-regions:
+              - ^docs/.*
+            black-list-labels:
```

Comment:
We can remove this. Doesn't make sense to have both a black list and a white list.